autoload/lsp/utils/location.vim

```diff
@@ -1,7 +1,12 @@
 function! s:open_location(path, line, col) abort
     normal! m'
     let l:buffer = bufnr(a:path)
-    if &modified && !&hidden
+    if exists('g:lsp_custom_open_location')
```

Comment:
Currently, `g:lsp_custom_open_location` is always defined in `plugin/lsp.vim`.
So this line should be `exists(...) && len()`.

Could you solve this problem?